Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs

```
@@ -93,7 +93,8 @@ private bool ActionKeywordMatch(Query query, Settings.ActionKeyword allowedActio
                 Settings.ActionKeyword.IndexSearchActionKeyword => Settings.IndexSearchKeywordEnabled &&
                                                                    keyword == Settings.IndexSearchActionKeyword,
                 Settings.ActionKeyword.QuickAccessActionKeyword => Settings.QuickAccessKeywordEnabled &&
-                                                                        keyword == Settings.QuickAccessActionKeyword
+                                                                        keyword == Settings.QuickAccessActionKeyword,
+                _ => false
```

I think we should throw an exception since it doesn't make sense here.